File: android/app/src/main/java/io/tenseventyseven/arion/ArionActivity.java
package io.tenseventyseven.arion;

import android.os.Bundle;

import com.getcapacitor.BridgeActivity;
import com.getcapacitor.Plugin;

// plugins
import com.nikosdouvlis.navigationbar.NavigationBar;

import java.util.ArrayList;

public class ArionActivity extends BridgeActivity {
  @Override
  public void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    this.init(savedInstanceState, new ArrayList<Class<? extends Plugin>>() {{
	  add(NavigationBar.class);
    }});
  }
}

File: android/app/src/main/kotlin/io/tenseventyseven/arion/ArionActivity.kt
package io.tenseventyseven.arion

import android.os.Bundle
import com.getcapacitor.BridgeActivity
import com.getcapacitor.Plugin
import java.util.ArrayList

// plugins
import com.nikosdouvlis.navigationbar.NavigationBar

class ArionActivity:BridgeActivity() {
  fun onCreate(savedInstanceState:Bundle) {
    super.onCreate(savedInstanceState)
    this.init(savedInstanceState, object:ArrayList<Class<out Plugin>>() {
      init{
        add(NavigationBar::class.java!!)
      }
    })
  }
}